.github/workflows/gnerate-extension-bundles.yml

```diff
@@ -0,0 +1,36 @@
+name: Generate extension bundles
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  generate-extension-bundles:
+    name: Generate extension bundles
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Use Node.js version from .nvmrc
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          cache: yarn
+
+      - run: yarn install --immutable | grep -v 'YN0013' # as in the polkadot workflows
+
+      - run: yarn download-chains-metadata
+      - run: yarn polkadot-dev-build-ts
+      - run: yarn set-signer-version --version=${{ github.ref_name }}
+      - run: yarn build:zip
+
+      - uses: actions/upload-artifact@v3
+        with:
+          name: signer-extension-bundles-v${{ github.ref_name }}
+          retention-days: 30
```

Comment:
I think it would be best to store them forever. Changes to signer might not be that frequent. If 30 days have passed since last release how would we revert the release?